app/actions/definitions/teams.tsx

```diff
@@ -1,40 +1,64 @@
+import { PlusIcon } from "outline-icons";
 import * as React from "react";
 import styled from "styled-components";
+import TeamNew from "~/scenes/TeamNew";
 import { createAction } from "~/actions";
 import { loadSessionsFromCookie } from "~/hooks/useSessions";
 
-export const changeTeam = createAction({
-  name: ({ t }) => t("Switch team"),
+export const switchTeamList = getSessions().map((session) => {
+  return createAction({
+    name: session.name,
+    section: "Switch Team",
```

Comment:
Needs defining in `actions/sections` and using the translation helper, you'll see from the ones there.

As discussed earlier and suggested by you last week lets switch to using "Workspaces" as the terminology as it better represents what's going on

Reply:
sorry, took care of this already but forgot to push!